src/store/reducers/executeQuery.ts

```diff
@@ -257,8 +277,24 @@ export const executeQueryApi = api.injectEndpoints({
                     }
 
                     dispatch(updateQueryInHistory(queryStats, queryId));
+                    dispatch(
+                        setQueryResult({
+                            type: ResultType.EXECUTE,
+                            data,
+                            isLoading: false,
+                            queryId,
+                        }),
+                    );
                     return {data};
```

Comment:
Let's not duplicate results and return empty data (`{data: null}`) here and in `explainQuery`.